pkgs/development/libraries/kissfft/default.nix

```diff
@@ -30,6 +30,11 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace "gcc" "${stdenv.cc.targetPrefix}cc" \
+      --replace "ar" "${stdenv.cc.targetPrefix}ar"
```

Comment:
Alternatively, as of v131.1.0 these aren't hardcoded.

https://github.com/mborgerding/kissfft/commit/fcf9fa9a47c4c0fec252bbb3951ce044ca8dc129

Reply:
Can this be fetched with fetchpatch?